src/Models/Attribute/AttributeGroupLang.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Flooris\Prestashop\Models\Attribute;
+
+use Flooris\Prestashop\Models\PrestashopModel;
+use Flooris\Prestashop\Traits\CompositeKeyModelTrait;
+
+/**
+ * Class AttributeGroupLang
+ *
+ * @property int    $id_attribute_group
+ * @property int    $id_lang
+ * @property string $name
+ * @property string $public_name
+ *
+ * @package Flooris\Prestashop\Models\Attribute
+ */
+class AttributeGroupLang extends PrestashopModel
+{
+    use CompositeKeyModelTrait;
+
+    /**
+     * Indicates if the model should be timestamped.
+     *
+     * @var bool
+     */
```

Comment:
we should remove these type of comment blocks, since the property are self exploratory